src/ripple/protocol/TER.h

```diff
@@ -305,7 +305,9 @@ class TERSubset
 
     // Trait tells enable_if which types are allowed for construction.
     template <typename T, typename = std::enable_if_t<Trait<T>::value>>
-    constexpr TERSubset (T rhs) : code_ ( TERtoInt (rhs)) { }
+    constexpr TERSubset (T rhs)
+    : code_ ( TERtoInt (rhs))
```

Comment:
Nit: There's still the extra space here. (I really don't care, but it's not how we normally format this).

Reply:
Just to be clear before I change anything, you're referring to the space between the paren and `TERtoInt`?

Reply:
Yes. I think most of us are writing `code_(TERtoInt(rhs))` these days, tho I think the style guide officially says it should be `code_ (TerToInt (rhs))`. Either way if fine with me, but yeah, we should get rid of the space between the opening paren and parameter for sure.